### PR TITLE
Fix helm installation issue

### PR DESCRIPTION
**Pull request by Divyansh42**
Earlier executable helm was not available as an archive,
now Helm latest version's excutable is available as an archive.
Also helm executable doesn't match with the client name, so
fixed that as well.

Fixes: #8 

Signed-off-by: divyansh42 <diagrawa@redhat.com>

### CODE REVIEW

**Review by tetchel — CHANGES_REQUESTED** [review]
This fix is in the right area and and I see that it works. However, I think there is an easier way.

In the caller, we already have `file.archiveFileName`, which this ends up being the same as in our helm case. Maybe instead of relying on `${client}-${os}-${arch}` working, we can just remove the file extension and add that to the glob pattern.

So the glob in the helm case would look for `helm` or `helm-linux-amd64`.
Note that `ghGlob.createGlob` accepts multiple patterns separated by newline.

File: src/installer/install.ts [hunk 1]
```diff
@@ -111,8 +113,19 @@ async function getExecutableTargetPath(file: ClientFile): Promise<string> {
...
-function getExecutable(file: ClientFile): string {
-    return getOS() === "windows" ? `${file.clientName}.exe` : file.clientName;
+function getExecutable(file: ClientFile, extractedDir?:string | undefined): string {
+    // get executable for other OS except Windows
+    let executable: string;
+    if (!extractedDir) {
+        executable = `${file.clientName}`;
+    }
+    else {
+        executable = fs.existsSync(`${extractedDir}/${file.clientName}`)
```
[the remark this answers is not in the document]
Comment (tetchel) [reply src/installer/install.ts:123]: `fs.promises` is what I use these days

**Comment by Divyansh42** [comment]
> This fix is in the right area and and I see that it works. However, I think there is an easier way.
> 
> In the caller, we already have `file.archiveFileName`, which this ends up being the same as in our helm case. Maybe instead of relying on `${client}-${os}-${arch}` working, we can just remove the file extension and add that to the glob pattern.
> 
> So the glob in the helm case would look for `helm` or `helm-linux-amd64`.
> Note that `ghGlob.createGlob` accepts multiple patterns separated by newline.

I think this approach really makes sense.
I don't think `archiveFileName` approach would be of much use.
So, I have added multiple patterns in the glob to match for `helm` or `helm-linux-amd64`

**Comment by tetchel** [comment]
looks great - 

can you change the `helm` version in the test workflow to `latest` or `*` so we can make sure it works?

**Review by tetchel — APPROVED** [review]

**Pull request merged**